service/s3/api_client.go

```diff
@@ -360,3 +360,7 @@ func addResponseErrorMiddleware(stack *middleware.Stack) {
 func addMetadataRetrieverMiddleware(stack *middleware.Stack) {
 	s3shared.AddMetadataRetrieverMiddleware(stack)
 }
+
+func handleResponseErrorWith200Status(stack *middleware.Stack) {
+	s3cust.HandleResponseErrorWith200Status(stack)
+}
```

Comment:
Why is this needed if it's effectively just a function alias?

Reply:
Could call the `s3cust` helper directly without the additional function helper since there is no additional setup needed.